src/milia/api/organization.cljc

```diff
@@ -107,6 +108,17 @@
                  :suppress-4xx-exceptions? true
                  :as-map? true))))
 
+#?(:cljs
```

Comment:
is there ever a situation where you'd want to call this from CLJ? Looks we could push the compilation tag in to only wrap the `:callback` key and value, if we want to make part of this work in CLJ

Reply:
Not sure but i get your point. So something like this then?

```
(defn change-org-member-role
  "Change the role of an organization member"
  [member org-name event-chan]
  (let [data {:username (:username member)
              :role (:role member)}]
    (parse-http :put
                (make-url "orgs" org-name "members")
                :callback
                #?(:clj nil)
                #?(:cljs #(put! event-chan {:updated-member member}))
                :http-options
                #?(:clj   {:form-params data
                           :content-type :json})
                #?(:cljs {:json-params data})
                :as-map? true)))
```
Though am not sure of the clj value for the `:callback` key. I have also tested this and it works.

Reply:
yea was thinking something like that, but for CLJ not even including the `:callback` key

Reply:
Cool in that case then i will have to remove the clj test for this function since it's failing without this clj part `#?(:clj nil)`. Am not sure if their is another way of making the test it work without that part?

Reply:
Yea, you can't just remove it because then the map would be invalid (it would have an odd number of elements). If you remove and put `:callback` inside the `:cljs` does that work? If not another option would be using `apply` and appending to a list of args only in `:cljs`, but that's a little messier

Reply:
Adding `:callback` inside the `:cljs` does not work, i think using `apply` is the only option. I have made the change using it, i think it's not that bad :)